### ISSUE TITLE
can i deploy it on the windows

### ISSUE THREAD

**Issue by hhuhhu** [irisli/stellarterm#206]
i saw many .sh files and when i run npm start in windows, it report synaxerror

**Comment by hhuhhu** [comment]
i saw many .sh files and when i run npm start in windows, it report synaxerror

**Comment by irisli** [comment]
No, unfortunately I don't have official support for Windows. Sorry

**Comment by hhuhhu** [comment]
thank you for your answer, i can get it run on windows now,  port 3001 can show normally, but 3000 just show the background. i just relink file directory.js and buildDirectory.js to the dest file

**Comment by irisli** [comment]
Oh okay, so maybe it has to do with the symlinks. Thanks for letting me know

**Comment by AnsarMadiniBaba** [comment]
How did you relink file directory.js and buildDirectory.js to the dest file

**Comment by AnsarMadiniBaba** [comment]
@hhuhhu how did you relink file directory.js and buildDirectory.js to the dest file
